synapse/handlers/room.py

```diff
@@ -959,7 +967,10 @@ async def create_room(
                     ratelimit=False,
                     content=content,
                     new_room=True,
+                    prev_event_ids=[last_sent_event_id],
+                    depth=depth,
                 )
+                depth += 1
```

Comment:
The loop below here over `invite_3pid_list` will eventually call `update_membership` to invite the 3pids. I think we could also pass through `depth` and `last_sent_event_id` there too if we wanted?

(Though I'd expect that most `/createRoom` calls don't come with a list of invitees; I think the changes to the `send()` helper are likely to be most impactful.)